src/Components/mua-roles-table/MUARolesTable.js

```diff
@@ -305,7 +305,7 @@ const MUARolesTable = ({
               </button>
             </Td>
           </Tr>
-          <Tr isExpanded={row.isExpanded} borders={false}>
+          <Tr isExpanded={row.isExpanded} borders={'false'}>
```

Comment:
I don't think this attribute should be a string. Unless its a PF thing.

Reply:
I changed it because I got this error in the console 

![image](https://user-images.githubusercontent.com/16109803/184643620-e390314a-a1d3-4640-ad55-85df8f37e8d0.png)

Reply:
So I guess we can just remove it?

Reply:
Yeah I'm removing it